### ISSUE TITLE
Create an INSTRUCTION.md

### ISSUE THREAD

**Issue by abhishekpatel946** [abhishekpatel946/PassManager-pyScript#2]
Create an Instruction.md file to describe each instruction in a simple way to set up the script.

**Comment by Sp1r1t72** [comment]
i might be interested, could you give me the details

**Comment by abhishekpatel946** [comment]
sure!

**Comment by abhishekpatel946** [comment]
pls check the [LINK](https://github.com/abhishekpatel946/PassManager-pyScript/tree/master/Screens/instructions_img). I already uploaded photos in ascending order you can check the photos and create an `INSTRUCTION.md` file and write down the step by step process.

**Comment by Sp1r1t72** [comment]
> 
> 
> pls check the [LINK](https://github.com/abhishekpatel946/PassManager-pyScript/tree/master/Screens/instructions_img). I already uploaded photos in ascending order you can check the photos and create an `INSTRUCTION.md` file and write down the step by step process.
> 
> **if you've any doubt feel free to reach me out!**

I have one question, do i make a .md to explain the steps in the images?

**Comment by abhishekpatel946** [comment]
Yup! Sure you can create an INSTRUCTION.md and explain each steps also you add photos too for more explanation. 👍🏼

**Comment by Sp1r1t72** [comment]
ill try and get it done as soon as possible. 👌

**Comment by abhishekpatel946** [comment]
ok wait I'll share with you a pdf for each step.

**Comment by Sp1r1t72** [comment]
> 
> 
> ok, wait I'll share with you a pdf for each step.
> 
> [c8643c1](https://github.com/abhishekpatel946/PassManager-pyScript/commit/c8643c1d4ed5ac74cd9698f231b5453493050126)
> 
> In this commit, I already mentioned each step with images so you can refer to these to clear your doubt. If images links are broken you just click on the link and it will redirect to the images folder.
> 
> **NOTE:- If you test this script at your own side, it will be a great help for me**

ill go ahead and test it so i can understand
thnx for the help
